README.md

```diff
@@ -67,29 +67,38 @@ Depending on the [broker configuration](https://github.com/alphagov/paas-rds-bro
+| `preferred_maintenance_window` | String  | The weekly time range during which system maintenance can occur (*)
 
 (*) Refer to the [Amazon Relational Database Service Documentation](https://aws.amazon.com/documentation/rds/) for more details about how to set these properties
 
 #### Update
 
 Update calls support the following optional [arbitrary parameters](https://docs.cloudfoundry.org/devguide/services/managing-services.html#arbitrary-params-update):
 
-| Option                       | Type    | Description
-|:-----------------------------|:------- |:-----------
-| apply_at_maintenance_window  | Boolean | Specifies whether the modifications in this request and any pending modifications are asynchronously applied as soon as possible (default) or if they should be queued until the Preferred Maintenance Window setting for the DB instance (*)
-| backup_retention_period      | Integer | The number of days that Amazon RDS should retain automatic backups of the DB instance (between `0` and `35`) (*)
-| preferred_backup_window      | String  | The daily time range during which automated backups are created if automated backups are enabled (*)
-| preferred_maintenance_window | String  | The weekly time range during which system maintenance can occur (*)
+| Option                         | Type    | Description
+|:-------------------------------|:------- |:-----------
+| `apply_at_maintenance_window`  | Boolean | Specifies whether the modifications in this request and any pending modifications are asynchronously applied as soon as possible (default) or if they should be queued until the Preferred Maintenance Window setting for the DB instance (*)
+| `backup_retention_period`      | Integer | The number of days that Amazon RDS should retain automatic backups of the DB instance (between `0` and `35`) (*)
+| `preferred_backup_window`      | String  | The daily time range during which automated backups are created if automated backups are enabled (*)
+| `preferred_maintenance_window` | String  | The weekly time range during which system maintenance can occur (*)
+| `preferred_maintenance_window` | String  | The weekly time range during which system maintenance can occur (*)
+| `reboot`                       | Boolean | Reboot the instance immediately. Any other parameter or change in the updated would be ignore.
+| `force_failover`               | Boolean | For HA failover during reboot. Only valid when used with `reboot` and for HA plans.
 
 (*) Refer to the [Amazon Relational Database Service Documentation](https://aws.amazon.com/documentation/rds/) for more details about how to set these properties
 
+#### Reboot
+
+Reboot is performed by passing the custom parameter `{ "reboot": true }` in an update. Pass `{ "reboot": true, "force_failover": true }` to force failover in a HA instance.
+
+It will not update the instance, so the new plan or custom parameters would be ignore.
```

Comment:
ignore -> ignored.

Reply:
Also we could mention that any pending minor version upgrades would be still applied.